Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -566,3 +575,23 @@ body[dir=rtl] [dir=ltr] .go-to-arrow {
         --checkerboard-dark-square: hsl(0, 0%, 5%);
     }
 }
+
+/* Keep this media query in sync with Main.js */
```

Comment:
Please explicitly mention the constant name in the comment `WI.NarrowLayoutWidth`
It makes it easier when doing a global project search to find references.

Reply:
Done.